### PR TITLE
Make CI pass

### PR DESCRIPTION
**Pull request by zunda-arrow**
Kinda don't know how to test this ngl

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -2,10 +2,10 @@
     "name": "piston-api",
     "version": "3.2.0",
     "description": "API for piston - a high performance code execution engine",
-    "module": "./dist/index.js",
+    "module": "./dist/src/index.ts",
     "type": "module",
     "scripts": {
-        "api": "node ./dist/index.js",
```
Comment (Enderchief) [inline api/package.json]: this can stay as it was before

**Review by Enderchief — CHANGES_REQUESTED** [review]

**Comment by Enderchief** [comment]
revert adding crystal :skull:

**Comment by Enderchief** [comment]
It wont help because the CI runs using production.

**Comment by zunda-arrow** [comment]
> revert adding crystal skull

i will, I'm just trying to trigger the package build CI

**Pull request merged**